Mathlib/GroupTheory/Submonoid/Basic.lean

```diff
@@ -167,7 +167,7 @@ initialize_simps_projections Submonoid (toSubsemigroup_carrier → coe)
 
 initialize_simps_projections AddSubmonoid (toAddSubsemigroup_carrier → coe)
 
-@[to_additive, simp]
+@[to_additive, simp] -- do we intentionally not have this simp for the additive declaration?
```

Comment:
This question should be answered before merging or marked as a TODO.

Reply:
Pretty sure this is an oversight from https://github.com/leanprover-community/mathlib4/pull/1224, unless @ADedecker recalls otherwise? I've fixed for now.